### PR TITLE
[INFRA-1310] Snapshot dependencies must die!

### PR DESCRIPTION
**Pull request by jglick**
[INFRA-1310](https://issues.jenkins-ci.org/browse/INFRA-1310)

@reviewbybees

### CODE REVIEW

**Comment by jglick** [comment]
Hmm, I have no permissions to add [org-level webhooks](https://ci.jenkins.io/job/Stapler/computation/events), will try to add to just this repo…

**Comment by ghost** [comment]
This pull request originates from a [CloudBees](https://www.cloudbees.com/) employee. At CloudBees, we require that all pull requests be reviewed by other CloudBees employees before we seek to have the change accepted. If you want to learn more about our process please see [this explanation](https://github.com/reviewbybees/about#about-reviewbybees).

**Review by ghost — APPROVED** [review]
:bee:

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+pipeline {
+    agent {
```
Comment (jtnord) [inline Jenkinsfile:2]: doesn't this need a `logRoatator` setup to remove old builds?

File: pom.xml [hunk 2]
```diff
@@ -10,7 +10,7 @@
     <dependency>
       <groupId>org.kohsuke.stapler</groupId>
       <artifactId>stapler</artifactId>
-      <version>[1.140,)</version>
+      <version>1.237</version> <!-- ~ jenkins-core:1.625.3 -->
```
Comment (jtnord) [inline pom.xml:13]: for reference: [MNG-3092](https://issues.apache.org/jira/browse/MNG-3092)

File: Jenkinsfile (hunk 1 above)
Comment (jglick) [reply Jenkinsfile:2]: I guess.

**Comment by recampbell** [comment]
@reviewbybees  done

**Comment by batmat** [comment]
+1, just hit this issue because I had a local build of stapler 1.253-SNAPSHOT. Building something totally unrelated at first sight (using blueocean).

**Review by batmat — APPROVED** [review]

**Comment by jglick** [comment]
see https://github.com/stapler/stapler/pull/186

**Pull request merged**